src/main/rx/expresslrs_common.c

```diff
@@ -78,15 +80,17 @@ elrsRfPerfParams_t rfPerfConfig[][ELRS_RATE_MAX] = {
         {0, RATE_LORA_200HZ, -112, 4380, 3000, 2500, 600, 5000},
         {1, RATE_LORA_100HZ, -117, 8770, 3500, 2500, 600, 5000},
         {2, RATE_LORA_50HZ, -120, 17540, 4000, 2500, 600, 5000},
-        {3, RATE_LORA_25HZ, -123, 17540, 6000, 4000, 0, 5000}
+        {3, RATE_LORA_25HZ, -123, 17540, 6000, 4000, 0, 5000},
     },
 #endif
 #ifdef USE_RX_SX1280
     {
-        {0, RATE_LORA_500HZ, -105, 1665, 2500, 2500, 3, 5000},
-        {1, RATE_LORA_250HZ, -108, 3300, 3000, 2500, 6, 5000},
-        {2, RATE_LORA_150HZ, -112, 5871, 3500, 2500, 10, 5000},
-        {3, RATE_LORA_50HZ, -115, 10798, 4000, 2500, 0, 5000}
+        {0, RATE_FLRC_1000HZ, -104, 389, 2500, 2500, 0, 5000},
```

Comment:
RADIO_TYPE_? is missing here?

Reply:
`RADIO_TYPE_` is in `elrsModSettings_t`, this is `elrsRfPerfParams_t`